bitnami/matomo/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: matomo
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/matomo
-version: 3.2.2
+version: 3.2.3
```

Here we are adding a new feature, according to [semver](http://semver.org) we should bump the minor coordinate:
```suggestion
version: 3.3.0
```